frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/ResetBranchToRemoteInfoDialog.kt

```diff
@@ -1,13 +1,12 @@
 package com.virtuslab.gitmachete.frontend.actions.dialogs
 
 import com.intellij.ide.util.PropertiesComponent
-import com.intellij.openapi.ui.DialogWrapper
+import com.intellij.openapi.ui.DoNotAskOption
 import com.intellij.openapi.ui.Messages
 import com.virtuslab.gitmachete.frontend.actions.base.BaseResetToRemoteAction.RESET_INFO_SHOWN
 import com.virtuslab.gitmachete.frontend.resourcebundles.GitMacheteBundle
 
-@Suppress("DEPRECATION")
-class ResetBranchToRemoteInfoDialog : DialogWrapper.DoNotAskOption.Adapter() {
+class ResetBranchToRemoteInfoDialog : DoNotAskOption.Adapter() {
```

Comment:
Does it change the behavior in any way? or is it just an equivalent replacement?

Reply:
Equivalent, the deprecation advises to use this `DoNotAskOption.Adapter` instead of `DialogWrapper.DoNotAskOption.Adapter`